fmriprep/workflows/anatomical.py

```diff
@@ -129,13 +129,21 @@ def bidsinfo(in_file):
             name='BIDSInfo',
             run_without_submitting=True)
 
+        pre_recon = pe.Node(
+            freesurfer.ReconAll(
+                directive='autorecon1',
+                flags='-nomotioncor -notalairach -nonuintensitycor '
+                      '-nonormalization -noskullstrip',
+            ),
+            name='PreRecon')
```

Comment:
There's no good way to run `recon-all` in nipype without trying to run a part of it, so this just sets it to `autorecon1` and disables all of the sub-stages. Could push a new PR to nipype to make `directive` undefined by default.